File: init_db.py
#!/usr/bin/env python3
"""
Initialize the database
"""
import asyncio
from bot.database import init_db, init_default_monitored_subnets

async def main():
    print("Initializing database...")
    await init_db()
    print("Initializing default monitored subnets...")
    await init_default_monitored_subnets()
    print("Database initialized successfully!")

if __name__ == "__main__":
    asyncio.run(main())

File: src/bin/init_db.rs
use anyhow::Result;
use tao_bot::database;
use tracing::info;

#[tokio::main]
async fn main() -> Result<()> {
    tracing_subscriber::fmt()
        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
        .init();

    info!("Initializing database...");
    database::init_db().await?;
    info!("Initializing default monitored subnets...");
    database::init_default_monitored_subnets().await?;
    info!("Database initialized successfully!");

    Ok(())
}
